diff --git a/Adding Url handlers to Operating System/Linux/README.md b/Adding Url handlers to Operating System/Linux/README.md
new file mode 100644
index 0000000..93381db
--- /dev/null
+++ b/Adding Url handlers to Operating System/Linux/README.md	
@@ -0,0 +1,10 @@
+# Creating Custom URL Handlers on Windows:
+
+- The `Windows Url handler.md` file contains instructions to create custom URL Handlers on Windows.
+
+
+- The `Testing Windows URL Handler.cs` file contains a test code to Test the created custom URL 
+
+- The `Output.md` describes the Out put on invoking `Ide:"Hello%20World"` from  the Browser.
+
+ References : https://docs.microsoft.com/
